### PR TITLE
dev-python/jupyter_server: enable doc generation

### PR DESCRIPTION
**Pull request by arthurzam**

### CODE REVIEW

File: dev-python/black/black-21.12_beta0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Copyright 2020-2022 Gentoo Authors
```
Comment (mgorny) [inline dev-python/black/black-21.12_beta0-r1.ebuild:1]: Plz do PEP517 here while you're revbumping it.

File: dev-python/sphinxemoji/sphinxemoji-0.2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
```
Comment (mgorny) [inline dev-python/sphinxemoji/sphinxemoji-0.2.0.ebuild:6]: Any reason not to use PEP517 here?

**Review by mgorny — CHANGES_REQUESTED** [review]
Plz PEP517 all the new packages.

**Comment by arthurzam** [comment]
Thanks, done and merged!

**Pull request closed without merging**